pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -245,7 +245,7 @@ func prepareMigrationFlags(isBlockMigration bool, isUnsafeMigration bool, allowA
 	migrateFlags := libvirt.MIGRATE_LIVE | libvirt.MIGRATE_PEER2PEER
 
 	if isBlockMigration {
-		migrateFlags |= libvirt.MIGRATE_NON_SHARED_INC
+		migrateFlags |= libvirt.MIGRATE_NON_SHARED_DISK
```

Comment:
could you please add few lines in the commit message about why this flag is needed?

Reply:
@vladikr what is the difference between the two flags?

Reply:
> VIR_MIGRATE_NON_SHARED_DISK 	= 	64 	
> 
> Migrate full disk images in addition to domain's memory. By default only
> non-shared non-readonly disk images are transferred. The
> VIR_MIGRATE_PARAM_MIGRATE_DISKS parameter can be used to specify which
> disks should be migrated. This flag and VIR_MIGRATE_NON_SHARED_INC are
> mutually exclusive.
> 
> VIR_MIGRATE_NON_SHARED_INC 	= 	128 	
> 
> Migrate disk images in addition to domain's memory. This is similar to
> VIR_MIGRATE_NON_SHARED_DISK, but only the top level of each disk's
> backing chain is copied. That is, the rest of the backing chain is
> expected to be present on the destination and to be exactly the same as
> on the source host. This flag and VIR_MIGRATE_NON_SHARED_DISK are
> mutually exclusive.
> 

I think in our case it doesn't change much. I can switch it back to `MIGRATE_NON_SHARED_INC` - Right now, I just wanted to see if it makes any difference.